systemtests/tests/python-bareos/test_list_command.py

```diff
@@ -258,15 +268,49 @@ def test_list_jobs(self):
         for job in result["jobs"]:
             self.assertEqual(job["type"], "B")
 
-        result = director.call("list jobs jobtype=Backup")
+        result = director.call("list jobs jobtype=Restore")
         self.assertTrue(result["jobs"])
         for job in result["jobs"]:
-            self.assertEqual(job["type"], "B")
+            self.assertEqual(job["type"], "R")
 
         result = director.call("list jobs jobtype=B,R")
         self.assertTrue(result["jobs"])
+
+        backup_jobs = 0
+        restore_jobs = 0
+        for job in result["jobs"]:
+            if (job["type"] == "B"):
+              backup_jobs +=1
+            if (job["type"] == "R"):
+              restore_jobs +=1
+
+        self.assertTrue(backup_jobs >= 1)
+        self.assertTrue(restore_jobs >= 1)
+
+        # list jobs joblevel/level=X
+        result = director.call("list jobs joblevel=F")
+        self.assertTrue(result["jobs"])
         for job in result["jobs"]:
-            self.assertTrue(job["type"] == "B" or job["type"] == "R")
+            self.assertEqual(job["level"], "F")
+
+        result = director.call("list jobs level=I")
+        self.assertTrue(result["jobs"])
+        for job in result["jobs"]:
+            self.assertEqual(job["level"], "I")
```

Comment:
I would also add this additional test:
```
result2 = director.call("list jobs joblevel=I")
self.assertTrue(result2["jobs"])
self.assertTrue(result == result2)
```
To ensure that both queries result in the same thing.